staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config_test.go

```diff
@@ -619,7 +619,7 @@ func TestKMSPluginHealthz(t *testing.T) {
 					p.service = nil
 					p.l = nil
 					p.lastResponse = nil
-					p.keyID = kmsv2Probe.keyID
```

Comment:
cc @enj for review

Reply:
This is fine as long as the test passes - we simply want to ignore the `keyID` field during the diff.